src/argus_htmx/templates/htmx/incidents/_incident_table_row.html

```diff
@@ -0,0 +1,41 @@
+<tr
+        id="incident-{{ incident.pk }}-row"
+>
+    <td>
+        {% block bulk_operations %}
+            <p>X</p>
+        {% endblock bulk_operations %}
+    </td>
+    <td>
+        {% block incident_pk %}
+            {% include "htmx/incidents/_incident_pk.html" %}
+        {% endblock %}
+    </td>
+    <td>
+        {% block start_time %}
+            {% include "htmx/incidents/_incident_start_time.html" %}
+        {% endblock %}
+    </td>
+    <td>
+        {% block status %}
+            {% include "htmx/incidents/_incident_status.html" %}
+        {% endblock %}
+    </td>
+    <td>{{ incident.level }}</td>
```

Comment:
I think Géant uses colors in some other columns for this, not a separate column.